Source/FAQView.swift

```diff
@@ -462,27 +513,44 @@ class FAQViewCell: UITableViewCell {
     self.answerTextView.textColor = configuration.answerTextColor
     self.questionLabel.font = configuration.questionTextFont
     self.answerTextView.font = configuration.answerTextFont
+    
+    if let positionType = configuration.positionType {
+    setupTextAlignment(position: positionType)
+    addConstraintsDependOnPositionType(position: positionType)
```

Comment:
Can you please format the code to match existing indentation?

Reply:
Is what I did on last commit is what you asked for ? , or I understand it wrong ?